.gitignore

```diff
@@ -0,0 +1 @@
+*.jsonl
```

Comment:
We actually have a few `jsonl` files in git, which are _supposed_ to be tracked. I'm not sure how that plays with this entry here?

Reply:
I moved the .gitignore into the `cmd` subfolder, so it won't interfere with our testdata but will ignore output generated by the binaries